poll/poll.py

```diff
@@ -166,6 +176,25 @@ def can_vote(self):
             return True
         return False
 
+    def can_view_private_results(self):
+        """
+        Checks to see if the user has permissions to view private results.
+        This only works inside the LMS.
+        """
+        if HAS_EDX_ACCESS and hasattr(self.runtime, 'user') and hasattr(self.runtime, 'course_id'):
+            # Course staff users have permission to view results.
+            if False and has_access(self.runtime.user, 'staff', self, self.runtime.course_id):
+                return True
+            else:
+                # Check if user is member of a group that is explicitly granted
+                # permission to view the results through django configuration.
+                group_names = getattr(settings, 'XBLOCK_POLL_ADDITONAL_GROUPS_WITH_VIEW_RESULTS_PERMISSION', [])
```

Comment:
Good golly. Can we call this something less verbose? Maybe `XBLOCK_POLL_EXTRA_VIEW_GROUPS`?